cocos/scene-graph/prefab/prefab-info.ts

```diff
@@ -222,7 +222,7 @@ export class PrefabInfo {
 
     @serializable
     @type([TargetOverrideInfo])
-    public targetOverrides?: TargetOverrideInfo[];
+    public targetOverrides?: TargetOverrideInfo[] = [];
```

Comment:
This property can have no value. But when you change it this way, it's always an empty array
I'm not so sure that's right